recipes/pulp/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.6.8" %}
+
+package:
+  name: pulp
+  version: '{{ version }}'
+
+source:
+  sha256: cfecd6a78c2b08539dc29b7e8281f43d729b1089bdb5e491c3c0956aadbb3fc7
+  fn: 'PuLP-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/P/PuLP/PuLP-{{ version }}.tar.gz
+
+build:
+  entry_points:
+    - pulptest=pulp:pulpTestAll
+    - pulpdoctest=pulp:pulpDoctest
+  number: 0
```

Comment:
Should this package be `noarch: python`?

Reply:
I tried, but it does not build in that case...

Reply:
Seems that a platform specific binary for the lp-solving is used.